cmd/root.go

```diff
@@ -131,6 +131,12 @@ func init() {
 		klog.Exitf("Failed to bind show-errored-containers flag: %v", err)
 	}
 
+	findCmd.Flags().Uint16P("timeout", "t", 10, "When finding container images, the time in seconds before canceling the operation.")
```

Comment:
`uint8` would cap at less than 5 minutes, which feels within the realm of possibility for a poorly-behaved repo. On the other hand, I can't imagine anyone needing _more_ than 9 hours :)